src/ActivityStickyQueues/FileProcessingWorkflow.cs

```diff
@@ -0,0 +1,71 @@
+using Microsoft.Extensions.Logging;
+using Temporalio.Workflows;
+
+namespace TemporalioSamples.ActivityStickyQueues;
+
+[Workflow]
+public class FileProcessingWorkflow
+{
+    [WorkflowRun]
+    public async Task RunAsync(int maxAttempts = 5)
+    {
+        for (var attempt = 1; attempt <= maxAttempts; ++attempt)
+        {
+            try
+            {
```

Comment:
I know TypeScript does it (and arguably shouldn't), but I wouldn't do a for loop and try/catch for attempts. I would let the workflow fail, and if you wanted retry and attempts and such, I would set the retry policy workflow side.